core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.ts

```diff
@@ -16,28 +30,29 @@ import { DotIconModule } from '@dotcms/ui';
         DotExperimentsConfigurationItemsCountComponent,
         DotMessagePipeModule,
         DotIconModule,
+        DotDynamicDirective,
+        UiDotIconButtonModule,
         //PrimeNg
         CardModule,
-        ButtonModule
+        ButtonModule,
+        UiDotIconButtonTooltipModule,
+        DotExperimentsConfigurationVariantsAddComponent
     ],
     templateUrl: './dot-experiments-configuration-variants.component.html',
     styleUrls: ['./dot-experiments-configuration-variants.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DotExperimentsConfigurationVariantsComponent {
-    maxLength = MAX_VARIANTS_ALLOWED;
-    @Input()
-    trafficProportion: TrafficProportion;
+    currentStep = ExperimentSteps.VARIANTS;
+    currentStatus = Status;
+    maxVariantsAllowed = MAX_VARIANTS_ALLOWED;
+    defaultVariantId = DEFAULT_VARIANT_ID;
 
-    changeTrafficProportionType() {
-        // to implemented
-    }
-
-    viewVariant() {
-        // to be implemented
-    }
-
-    addNewVariant() {
-        // to be implemented
-    }
+    @Input() status: StepStatus;
+    @Input() variants: Variant[];
+    @Output() showSidebar = new EventEmitter<void>();
+    @Output() hiddenSidebar = new EventEmitter<void>();
```

Comment:
Can't you unify this to one event that you can emit with data to know if open or close